packages/builder/src/components/common/NavItem.svelte

```diff
@@ -53,11 +53,8 @@
     scrollApi.scrollTo(bounds)
   }
 
-  const getStyle = (indentLevel, selectedBy) => {
+  const getStyle = (indentLevel: number) => {
     let style = `padding-left:calc(${indentLevel * 14}px);`
-    if (selectedBy) {
-      style += `--selected-by-color:${helpers.getUserColor(selectedBy)};`
-    }
```

Comment:
Was this not required to be able to show if other users have resources selected?

Reply:
This did not work. `selectedBy` is an array of users, this will not work fine. The different coloring is already  handled in the `UserAvatar` component